src/registrar/templates/application_status.html

```diff
@@ -23,6 +23,7 @@ <h1>Domain request for {{ domainapplication.requested_domain.name }}</h1>
           {% elif domainapplication.status == 'in review' %} In Review 
           {% elif domainapplication.status == 'rejected' %} Rejected
           {% elif domainapplication.status == 'submitted' %} Submitted
+          {% elif domainapplication.status == 'ineligible' %} Ineligible
```

Comment:
Did you mean for this to be 'restricted'? Rather than 'ineligible'?

Reply:
Kind of along same lines as Zander......just to confirm: for users statuses, we are setting "restricted" and for domain applications or just related to application we are setting "ineligible", correct?

Reply:
Correct, so this should indeed say ineligible